familyconnect_registration/testsettings.py

```diff
@@ -12,3 +12,7 @@
 CELERY_ALWAYS_EAGER = True
 BROKER_BACKEND = 'memory'
 CELERY_RESULT_BACKEND = 'djcelery.backends.database:DatabaseBackend'
+
+PASSWORD_HASHERS = (
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+)
```

Comment:
I'm not clear on the necessity of this change. Does it just make the tests faster?